ui/app/styles/components/splash-page.scss

```diff
@@ -19,9 +19,7 @@
 .splash-page-container {
   margin: $size-2 0;
 }
+
 .splash-page-header {
   padding: $size-6 $size-5;
-}
-.splash-page-sub-header {
-  margin: 0 $size-5 $size-6;
-}
+}
```

Comment:
this is a nit, but i think you'll want that newline at the end :)

Reply:
I was trying VSCode out for this one, didn't realize that wasn't the default. Thanks for catching that!